src/languages/pt-BR.ts

```diff
@@ -546,6 +546,7 @@ const translations = {
         userID: 'ID do Usuário',
         disable: 'Desativar',
         export: 'Exportar',
+        basicExport: 'Exporta\u00E7\u00E3o b\u00E1sica',
```

Comment:
NAB we can do this in a follow up

```suggestion
        basicExport: 'Exportação básica',
```